pkg/storage/db.go

```diff
@@ -536,140 +299,26 @@ func resIDFromARN(ARN string) string {
-		for _, hostname := range changes.Hostnames {
-			err := db.insertNetworkChangeEvent(ctx, timestamp, true, isJoin, resourceID, val, &hostname, tx)
-			if err != nil {
-				return err
-			}
-		}
-	}
-
-	return nil
-}
-
-func (db *DB) insertIPAddress(ctx context.Context, ipAddress string, tx *sql.Tx) error {
-
-	// this lib won't give back the last INSERTed row ID, so we don't bother with `RETURNING ...`
-	// See https://stackoverflow.com/questions/34708509/how-to-use-returning-with-on-conflict-in-postgresql
-	if _, err := tx.ExecContext(ctx, fmt.Sprintf(`INSERT INTO %s VALUES ($1) ON CONFLICT DO NOTHING`, tableAWSIPS), ipAddress); err != nil { // nolint
-		return err
-	}
-
-	return nil
-}
-
-func (db *DB) insertHostname(ctx context.Context, hostname string, tx *sql.Tx) error {
-
-	// this lib won't give back the last INSERTed row ID, so we don't bother with `RETURNING ...`
-	// See https://stackoverflow.com/questions/34708509/how-to-use-returning-with-on-conflict-in-postgresql
-	if _, err := tx.ExecContext(ctx, fmt.Sprintf(`INSERT INTO %s VALUES ($1) ON CONFLICT DO NOTHING`, tableAWSHostnames), hostname); err != nil { // nolint
-		return err
-	}
-
-	return nil
-}
-
-func (db *DB) insertNetworkChangeEvent(ctx context.Context, timestamp time.Time, isPublic bool, isJoin bool, resourceID string, ipAddress string, hostname *string, tx *sql.Tx) error {
-	// Postgres 11 automatically propagates the parent index to child tables, so no need to
-	// explicitly create an index on the possibly created new table.
-
-	// this lib won't give back the last INSERTed row ID, so we don't bother with `RETURNING ...`
-	// See https://stackoverflow.com/questions/34708509/how-to-use-returning-with-on-conflict-in-postgresql
-	_, err := tx.ExecContext(ctx, fmt.Sprintf(`INSERT INTO %s VALUES ($1, $2, $3, $4, $5, $6)`, tableAWSEventsIPSHostnames), timestamp, isPublic, isJoin, resourceID, ipAddress, hostname) // nolint
-	return err
-}
-
 // FetchAll gets all the assets present at the specified time
 func (db *DB) FetchAll(ctx context.Context, when time.Time, count uint, offset uint, typeFilter string) ([]domain.CloudAssetDetails, error) {
-	return db.runQuery(ctx, bulkResourcesQuery, when, typeFilter, count, offset)
+	return nil, errors.New("bulk export API is not available")
 }
 
 // FetchByHostname gets the assets who have hostname at the specified time
 func (db *DB) FetchByHostname(ctx context.Context, when time.Time, hostname string) ([]domain.CloudAssetDetails, error) {
-	ver, err := db.getSchemaVersion(ctx)
-	if err != nil {
-		return nil, err
-	}
-	var assets []domain.CloudAssetDetails
-	if ver < ReadsFromNewSchemaVersion {
-		sqlstmt := fmt.Sprintf(latestStatusQuery, `aws_hostnames_hostname`)
-		assets, err = db.runQuery(ctx, sqlstmt, hostname, when)
-	} else {
-		assets, err = db.runFetchByIPQuery(ctx, false, resourceByHostnameQuery, hostname, when)
-	}
-	return assets, err
+	return db.runFetchByIPQuery(ctx, false, resourceByHostnameQuery, hostname, when)
```

Comment:
Nit: while we're in here, is it possible to refactor `runFetchByIPQuery`'s name since we're also searching by hostname with it?

Reply:
Fixed